src/helpers/literal.ts

```diff
@@ -0,0 +1,15 @@
+import { LiteralObject } from '../types/Literal'
+
+/**
+ * Literal is a tagged template for an object { value: string, escape: false }
+ * @example
+ * // literal`example_pkey` return { value: 'example_pkey', escape: false }
+ * @example
+ * // literal`['example_key01', 'example_key02']` return { value: '['example_key01', 'example_key02']', escape: false }
+ * @param {TemplateStringsArray} literalValue Literal value
+ * @return {LiteralObject} LiteralObject
+ */
+export const literal = (literalValue: TemplateStringsArray): LiteralObject => {
+  if (!literalValue || !literalValue[0]) throw new Error('literalValue cannot be null or empty')
+  return { value: literalValue[0], escape: false }
+}

```

Comment:
Can you please change this definition to a function definition to follow the code style?

```suggestion
function literal (literalValue: TemplateStringsArray): LiteralObject {
  if (!literalValue || !literalValue[0]) throw new Error('Value for the literal argument cannot be null or empty')
  return { value: literalValue[0], escape: false }
}

export literal
```

Reply:
Actually, I think there are empty arguments (as of #39), so probably, we would want something like this:

```suggestion
function literal (literalValue: TemplateStringsArray): LiteralObject {
  return { value: literalValue[0] || "", escape: false }
}

export literal
```

Reply:
Done.